### PR TITLE
LPS-171601 | Test Fix | Master

### PR DESCRIPTION
**Pull request by pedroamarall**
Ticket:
https://issues.liferay.com/browse/LPS-171601

Changes in these files:

AddCalendar.testcase
CalendarMyCalendars.macro
CalendarEditCalendar.path

### CODE REVIEW

**Comment by pedroamarall** [comment]
ci:test:sf

File: modules/apps/calendar/calendar-web-test/src/testFunctional/macros/CalendarMyCalendars.macro [hunk 1]
```diff
@@ -34,8 +34,8 @@ definition {
 
 	macro viewColorMyCalendars {
```
Comment (epark18) [inline modules/apps/calendar/calendar-web-test/src/testFunctional/macros/CalendarMyCalendars.macro:35]: I know you didn't make this macro, but I realized no other test uses it. We are trying to move away from having assertions in the macro level. It's okay to remove this macro in this case and move the assertion to your testcase.

**Review by epark18 — COMMENTED** [review]
Hey @pedroamarall nice job on this fix. Please consider my comment and let me know if something is not clear. 
One thing I do want to mention is to not forget to attach local test report after you run your test onto the ticket.

**Comment by pedroamarall** [comment]
ci:tes:sf

File: modules/apps/calendar/calendar-web-test/src/testFunctional/macros/CalendarMyCalendars.macro (hunk 1 above)
Comment (pedroamarall) [reply modules/apps/calendar/calendar-web-test/src/testFunctional/macros/CalendarMyCalendars.macro]: Hello @epark18 , thanks for your description, I changed now!

**Comment by pedroamarall** [comment]
ci:test:sf

**Comment by epark18** [comment]
ci:forward

**Comment by epark18** [comment]
ci:test:sf

**Pull request closed without merging**